### DISCUSSION TITLE
The language code of Korean

### DISCUSSION THREAD

**Discussion by jinhanpae** [quarto-dev/quarto-cli#4186] [Q&A]
There is a typo in the illustration of the lang code for Korean. (the "Document Language" tab)

![image](https://user-images.githubusercontent.com/122417903/216203098-79486aed-89ee-494d-b97d-4a1b28a8340c.png)

The language code of Korean is "ko", not "kr"

![image](https://user-images.githubusercontent.com/122417903/216202884-3e410884-824f-47c8-8ae0-905b397bf2a6.png)

You may also consider changing the file name of the Korean language translation from "_language-**kr**.yml" to "_language-**ko**.yml"

**Comment by cscheid** [comment]
Thanks for the report! This is a bug, tracking on #4187.
